mappings/maestro_file_centric.json

```diff
@@ -0,0 +1,239 @@
+{
+  "aliases": {
+    "file_centric": {}
+  },
+  "settings": {
+    "index.number_of_shards": 3,
+    "index.max_results_window": 300000,
+    "analysis": {
+      "analyzer": {
+        "autocomplete_analyzed": {
+          "filter": ["lowercase", "edge_ngram"],
+          "tokenizer": "standard"
+        },
+        "autocomplete_prefix": {
+          "filter": ["lowercase", "edge_ngram"],
+          "tokenizer": "keyword"
+        },
+        "lowercase_keyword": {
+          "filter": ["lowercase"],
+          "tokenizer": "keyword"
+        }
+      },
+      "filter": {
+        "edge_ngram": {
+          "max_gram": 20,
+          "min_gram": 20,
+          "side": "front",
+          "type": "edge_ngram"
+        }
+      }
+    }
+  },
+  "mappings": {
+    "dynamic": false,
+    "date_detection": false,
+    "properties": {
+      "file_autocomplete": {
+        "fields": {
+          "analyzed": {
+            "analyzer": "autocomplete_prefix",
```

Comment:
should this be `autocomplete_analyzed`? see here: https://github.com/NCI-GDC/gdc-models/blob/89e5a052d0bd8b39671d3cec941f4cd3e2c18c90/es-models/gdc_from_graph/file.mapping.yaml#L2362-L2374

Reply:
i copied @blabadi settings...Bashar thoughts?

Reply:
Will update

Reply:
updated, however I need to take a deeper look on what analyzers to use this was built on previous suggestions, I'll need dedicated time to try different analyzers and verify that they will work as desired

Reply:
@junjun-zhang for my infromation what do you use the `prefix analyzer` for?
and in the link you posted there is a `clinical_normalizer`  should I also look into adding that too ?

Reply:
We spent a lot of time tweaking these settings back in the day. Don't remember all the details, but `prefix analyzer` will help you find `ABC-1234` and rank it to the top when you type in `abc-1`, otherwise, `ABC` will be ranked higher. Thanks to the difference between `tokenizer: standard` and `tokenizer: keyword`. Yes, same should be set for `clinical_normalizer`

Reply:
@junjun-zhang @blabadi I think we have another ticket to reverify the analyzer needs. 

@blabadi We can test analyzers with the IHCC demo i believe!